client/me/purchases/manage-purchase/purchase-meta.jsx

```diff
@@ -221,13 +224,72 @@ function PurchaseMetaOwner( { owner } ) {
 	);
 }
 
+const formatPurchasePrice = ( price, currency ) =>
+	formatCurrency( price, currency, {
+		isSmallestUnit: true,
+		stripZeros: true,
+	} );
+
+function getDIFMPriceDetails( purchase ) {
+	const [ tier0, tier1 ] = purchase.priceTierList;
+	const perExtraPagePrice = tier1.minimumPrice - tier0.minimumPrice;
+
+	const { maximumUnits: noOfBaselinePagesIncluded, minimumPriceDisplay: formattedOneTimeFee } =
+		tier0;
+	const { purchaseRenewalQuantity: noOfPages, currencyCode } = purchase;
+	const extraPageCount = noOfPages - noOfBaselinePagesIncluded;
+	const costOfExtraPages = extraPageCount * perExtraPagePrice;
+	const formattedCostOfExtraPages = formatPurchasePrice( costOfExtraPages, currencyCode );
+	return {
+		extraPageCount,
+		costOfExtraPages: formattedCostOfExtraPages,
+		oneTimeFee: formattedOneTimeFee,
+	};
+}
+
 function PurchaseMetaPrice( { purchase } ) {
 	const translate = useTranslate();
+	const productsList = useSelector( getProductsList );
 	const { productSlug, productDisplayPrice } = purchase;
 	const plan = getPlan( productSlug ) || getProductFromSlug( productSlug );
 	let period = translate( 'year' );
 
 	if ( isOneTimePurchase( purchase ) || isDomainTransfer( purchase ) ) {
+		if ( isDIFMProduct( purchase ) ) {
+			const { extraPageCount, costOfExtraPages, oneTimeFee } = getDIFMPriceDetails(
+				purchase,
+				productsList
+			);
+			return (
+				<div>
+					<div>
+						{ translate( 'Service : %(oneTimeFee)s {{period}}(one-time){{/period}}', {
+							args: {
+								oneTimeFee,
+							},
+							components: {
+								period: <span className="manage-purchase__time-period" />,
+							},
+						} ) }
+					</div>
+					<div>
+						{ translate(
+							'%(extraPageCount)d extra pages : %(costOfExtraPages)s {{period}}(one-time){{/period}}',
```

Comment:
We should also add singular/plural versions here, similar to the ones added in `client/me/purchases/manage-purchase/purchase-meta.jsx`.